bootable-jar/WFLY-13305_Bootable_Jar.adoc

```diff
@@ -0,0 +1,469 @@
+= WFLY-13305: WildFly bootable JAR
+:author:            Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+We are proposing a way to package server and application inside a bootable JAR. The JAR contains everything needed to run the server.
+The packaged server can benefit from Galleon tooling and be trimmed down to user needs (thanks to Galleon layers).
+Such executable JAR can then be run on bare metal and in cloud execution contexts.
+
+A simple example of a https://github.com/wildfly-extras/wildfly-jar-maven-plugin/blob/master/examples/jaxrs/pom.xml[pom file] to create a WildFly bootable JAR.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13305[WFLY-13305]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1385[EAP7-1385]
```

Comment:
Should this be https://issues.redhat.com/browse/EAP7-1538 or should that JIRA be added here?

Reply:
@jamezp EAP7-1538 is an addition. EAP7-1385 is related to WFLY-13305